docs/fidesops/Dockerfile.docs

```
@@ -1,4 +1,4 @@
-FROM python:3.8-slim-buster
+FROM python:3.8-slim-bullseye
```

Let's go ahead and harmonize the Python version also.

```suggestion
FROM python:3.9-slim-bullseye
```
```